### PR TITLE
Submitting /guide once again. 

### PR DESCRIPTION
**Pull request by Specifer**
Ping me if it shows any error, My local server is showing some streamer error currently.

### CODE REVIEW

**Review by RussellLVP — CHANGES_REQUESTED** [review]
Hi Specifer,

Thank you! I think this is fine if you could use the `//features/commands/commands.js` infrastructure for this, like you have before :) That makes it much easier too!

Thanks,
Russell

**Comment by Specifer** [comment]
Ok, thank you for saying!

**Comment by Specifer** [comment]
@RussellLVP, Done!

**Comment by RussellLVP** [comment]
Why are you deleting `account_commands.js`? That certainly shouldn't be deleted! Looks good otherwise.

File: javascript/features/account/account_commands.js [hunk 1]
```diff
@@ -58,7 +58,7 @@ export class AccountCommands {
     //
     // Opens the account management flow for either the |player| or |targetPlayer|, when set. The
     // availability of options is dependent on server configuration, as well as the player's rights
-    // on the server in general. Certain options are limited to VIPs.
+    // on the server in general. Certain options are limited to VIPs like alias feature.
```
Comment (Specifer) [inline javascript/features/account/account_commands.js:61]: I had to do it since to bring it to github desktop, Hope you understand :D.

**Comment by Specifer** [comment]
Did some changes in Onplayercommand.pwn :D.

**Comment by Specifer** [comment]
@RussellLVP, Fixed the error. Feel free to merge.

File: pawn/Resources/Callbacks/OnPlayer/OnPlayerCommandText.pwn [hunk 2]
```diff
@@ -353,6 +353,9 @@ public OnPlayerCommandText(playerid, cmdtext[]) {
         return 0;
     }
 
+    if(strcmp(cmd, "/guide", true) == 0 || !Player(playerid)->isAdministrator() || !Player(playerid)->isManagement())
```
Comment (RussellLVP) [inline pawn/Resources/Callbacks/OnPlayer/OnPlayerCommandText.pwn]: Please undo these changes, they're not necessary.

**Comment by RussellLVP** [comment]
Thank you! This is good to go if you could remove the two lines from the Pawn code, they're not necessary.

**Comment by Specifer** [comment]
@RussellLVP, Done!

**Review by RussellLVP — APPROVED** [review]

**Pull request merged**